torch/distributed/nn/functional.py

```diff
@@ -101,6 +101,25 @@ def reduce_scatter(output, input_list, op=ReduceOp.SUM, group=group.WORLD):
     return _Reduce_Scatter.apply(op, group, output, *input_list)
 
 
+def _reduce_scatter_base(output, input, op=ReduceOp.SUM, group=group.WORLD):
```

Comment:
I think we should try to call the new functional collective instead of using the old functional collective?

Reply:
This is for control group which is to mimic Megatron.